index.html

```diff
@@ -43,23 +44,24 @@ <h1>Books</h1>
         <br>
         <br>
         <br>
+        <!--Button ADD, CLEAR FORM-->
         <button type="button" onclick="addBook();">Add</button>
         <button type="button" onclick="cleanFormInput();">Clear</button>
     </form>
 </div>
-
+      <!--Sorry-->
     <div id="book-save" style="visibility: hidden;"class="save-book save-book-monitor">
         <h2 class="text-danger-header">Sorry!</h2>
         <p class="text-danger"><strong>The book has already existed.</strong></p>
         <a id="book-save-click" href="#" class="button-book-save" value="" onclick="cleanButton();">Undestand!</a>
     </div>
- 
+    <!--Sucsess-->
     <div id="book-bd" style="visibility: hidden;"class="save-bd save-b-d">
```

Comment:
```suggestion
    <div id="book-save-success" style="visibility: hidden;"class="save-success save-b-d">
```
change style name accordinly